src/main/resources/db/migration/V1_15__service_provider.sql

```diff
@@ -0,0 +1,7 @@
+create table service_provider (
+  id varchar(30) not null,
+  name text not null,
+  email_address text,
```

Comment:
i think we should more tightly scope what the email is e.g. incoming_referral_distribution_email or something

Reply:
renamed